writer_test.go

```diff
@@ -0,0 +1,53 @@
+package cpio_test
+
+import (
+	"bytes"
+	"io"
+	"os"
+	"testing"
+
+	cpio "github.com/cavaliercoder/go-cpio"
```

Comment:
I think the prefix is redundant. It should resolve to the package name which is already `cpio`.

Reply:
Indeed. goimports added this import for me, but removing the cpio indeed works.